### PR TITLE
Enable compatibility with Sway

### PR DESCRIPTION
**Pull request by Synthetica9**
Switched to i3ipc instead of i3-py, the last update to i3-py was in 2012, i3ipc is actively developed. This also enables compatibility with Sway.

### CODE REVIEW

**Comment by tmfink** [comment]
Thanks for the PR! I'll add some comments.

File: i3-wk-switch.py [hunk 1]
```diff
@@ -10,10 +10,10 @@
 import logging
 import sys
 from pprint import pformat
+import i3ipc
```
Comment (tmfink) [inline i3-wk-switch.py]: Move this 3rd party import after time, with a newline in between (where the `import i3` used to be).

File: i3-wk-switch.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/python
+#!/usr/bin/env python
 
-"""Emulates xmonad's workspace switching behavior in i3"""
+"""Emulates xmonad's workspace switching behavior in i3 and sway"""
 
 # pylint: disable=no-member,invalid-name
```
Comment (tmfink) [inline i3-wk-switch.py]: Add `too-many-function-args` to suppress warning

File: i3-wk-switch.py [hunk 3]
```diff
@@ -67,15 +67,15 @@ def get_workspace(num):
...
     """Switches to workspace number"""
-    i3.workspace('number %d' % num)
+    i3.command('workspace %d' % num)
 
 
 def swap_visible_workspaces(wk_a, wk_b):
     """Swaps two workspaces that are visible"""
     switch_workspace(wk_a['num'])
-    i3.command('move', 'workspace to output ' + wk_b['output'])
+    i3.command('move workspace to output ' + wk_b['output'])
```
Comment (tmfink) [inline i3-wk-switch.py]: In hindsight, these commands should use string formatting with `%` as above.

File: README.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
+requested workspace to the current monitor ("output" in i3's terminology). If
+the requested workspace is showing on another monitor, then the current
+workspace and requested workspace will swap positions with each other.
 
 
 # Requirements
 
 - [i3 window manager](https://i3wm.org/)
-- [i3-py](https://github.com/ziberna/i3-py)
+- [i3ipc](https://github.com/acrisci/i3ipc-python)
```
Comment (tmfink) [inline README.md]: 1. Please describe this as "i3ipc python module"
2. Add a sub bullet "Install with `pip install i3ipc`".

**Review by tmfink — CHANGES_REQUESTED** [review]
Looks really good! Your changes also brought to my attention changes I should have already made.

File: i3-wk-switch.py (hunk 2 above)
Comment (Synthetica9) [reply i3-wk-switch.py]: ~~I don't get a warning?~~

NVM, I see it now. Fixing.

**Comment by Synthetica9** [comment]
@tmfink Everything should be okay now.

**Review by tmfink — APPROVED** [review]

**Pull request merged**